db/models.py

```diff
@@ -14,3 +14,43 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(blank=True)
+    actors = models.ManyToManyField(Actor, related_name="actor")
```

Comment:
The `related_name` 'actor' for the `actors` field might cause confusion if there are multiple actors related to a movie. Consider using a plural form like 'actors'.